capa/main.py

```diff
@@ -493,7 +495,7 @@ def main(argv=None):
     try:
         taste = get_file_taste(args.sample)
     except IOError as e:
-        logger.error("%s", str(e))
+        logger.error("%s", e.args[0])
```

Comment:
what does it mean to fetch the first element of `args` here? does this cover all possible cases?

Reply:
this post seems relevant, but it doesnt work:
https://stackoverflow.com/a/18413418/87207

Reply:
as far as i can tell, there's not a programmatic way to render the IOError with non-ASCII filename unless we handle the IOError separately and reach into the args, like you've done here:
![Uploading image.png…]()

Reply:
so.... i've come to the same conclusion as you. picking `e.args[0]` seems like the best option. lets just document our work here.